### ISSUE TITLE
Search crashes the entire page

### ISSUE THREAD

**Issue by bladeSk** [Vincent-P/firefox-better-history#39]
When I type a single letter into the search field, the page goes blank and this error is thrown: `TypeError: can't access property "toLowerCase", visit.title is null`

https://github.com/Vincent-P/firefox-better-history/blob/65d10e1063faae396c98fa18d8a4980763f7d468/src/app.js#L84

`visit.title` can be `null` under certain circumstances. It may have something to do with me having imported history from Chrome, nevertheless, this can happen and renders the search function unusable. The fix is easy - `visit.title.toLowerCase()` -> `(visit.title || '').toLowerCase()`

**Comment by nullstalgia** [comment]
> It may have something to do with me having imported history from Chrome

I don't think so. It started happening to me around the same time, and I just saw a review on the Firefox app store saying searching crashes their page too. Maybe try to send in a PR?

**Comment by totoroot** [comment]
I can confirm that the search functionality recently broke completely. Running different, relatively recent versions of Firefox on both macOS and Linux and getting a blank page once I enter a single character in the search bar.

**Comment by mooms06** [comment]
I concur, the extension is broken on latest Firefox stable.
